lib/registries/addon/components/sharing-icons/sharing-anchor/component.ts

```diff
@@ -0,0 +1,5 @@
+import { tagName } from '@ember-decorators/component';
+import Component from '@ember/component';
+
+@tagName('')
```

Comment:
When making a component tagless, always remember to add `...attributes` to the main element in the template. You never know whether someone will need to add attributes in the future, and don't want to make that future dev have to dig into the thing they're using.